src/main/java/searchengine/vsm/VectorSpaceModel.java

```diff
@@ -69,9 +77,17 @@ private void calculateTFIDF(List<TFIDFDocument> documents) {
     for (TFIDFDocument document : documents) {
       HashMap<String, Double> innerMap = new HashMap<>();
 
+      for (String term : document.getTF().keySet()) {
+        int termFrequency = document.getTF().get(term);
+        if (termFrequency > maximumFrequency) maximumFrequency = termFrequency;
+      }
+
       for (String term : document.getTF().keySet()) {
         // TFIDF_td = TF_td * IDF_t
         // td: term in document, t: term
+        //This is normalizing to avoid bias for large documents
+        //innerMap.put(term, 0.5 + (0.5 * document.getTF().getOrDefault(term, 0) / (maximumFrequency * document.getTF().getOrDefault(term, 0))) * idf.getOrDefault(term, 1.0));
```

Comment:
Er der en grund til at det her er udkommenteret? Skal det slettes?

Reply:
Hvis vi sletter noget formel noget fra det tidligere, har vi så tjekket op i forhold til teorien hvad der er korrekt at gøre?

Reply:
Det er det gamle, vi vil gerne beholde det for reference

Reply:
```suggestion
        //OLD VERSION, KEPT FOR REFERENCE: innerMap.put(term, 0.5 + (0.5 * document.getTF().getOrDefault(term, 0) / (maximumFrequency * document.getTF().getOrDefault(term, 0))) * idf.getOrDefault(term, 1.0));
```